source/js/main.js

```diff
@@ -0,0 +1,3 @@
+window.globalVariables = require("./global-variables");
```

Comment:
Maybe we should put all these variables inside of a single path, like `window.styleguide.cookieAlert` and `window.styleguide.globalVariables` for example. Doing so, it's easier to see which variables are coming from the styleguide on the portal side, and it is also a way to avoid them from being overridden by third-party scripts. What do you think?

Reply:
Hey, thanks for thinking about this.
In my opinion I think it is not good. 
First, cookieAlert has actually javascript logic and should be its own component, so this should be outside of this discussion, so let's talk about the global-variables.
Second, the component global-variables could be used for general variables specifically used for Javascript and logics.

Reply:
Right. I was just thinking that if you go to the console in the main page of the portal and try to check what is inside of the `window` variable, a lot of variables will be returned as output and due to that, it is difficult to see which ones are coming from the styleguide or third-party extensions. Maybe we should have a way to check that they're all coming from the same origin.